airflow/dag_processing/bundles/base.py

```diff
@@ -84,7 +87,9 @@ def path(self) -> Path:
         """
         Path for this bundle.
 
-        Airflow will use this path to load/execute the DAGs from the bundle.
+        Airflow will use this path to find/load/execute the DAGs from the bundle.
+        You can change this path during refreshes, but it cannot change between refreshes.
+        It is also should be retrievable once the bundle has been initialized.
```

Comment:
retrievable? not sure what that means.  i think you mean, the dags should ... be "there" after you hit initialize?

Reply:
maybe "after calling initialize, all dag files in the bundle should be accessible from this path and its subdirectories" or something